src/diagnostics/default_diagnostics.jl

```diff
@@ -0,0 +1,194 @@
+    common_diagnostics(24 * 60 * 60, max, output_writer, short_names)[1]
+
+"""
+    daily_mins(short_names...; output_writer = HDF5Writer())
+
+
+Return a list of `ScheduledDiagnostics` that compute the daily min for the given variables.
+"""
+daily_mins(short_names...; output_writer = HDF5Writer()) =
+    common_diagnostics(24 * 60 * 60, min, output_writer, short_names...)
+"""
+    daily_min(short_names; output_writer = HDF5Writer())
+
+
+Return a `ScheduledDiagnostics` that computes the daily min for the given variable.
+"""
+daily_min(short_names; output_writer = HDF5Writer()) =
+    common_diagnostics(24 * 60 * 60, min, output_writer, short_names)[1]
+"""
+    daily_averages(short_names...; output_writer = HDF5Writer())
+
+
+Return a list of `ScheduledDiagnostics` that compute the daily average for the given variables.
+"""
+# An average is just a sum with a normalization before output
+daily_averages(short_names...; output_writer = HDF5Writer()) =
+    common_diagnostics(
+        24 * 60 * 60,
+        (+),
+        output_writer,
+        short_names...;
+        pre_output_hook! = average_pre_output_hook!,
+    )
+"""
+    daily_average(short_names; output_writer = HDF5Writer())
+
+
+Return a `ScheduledDiagnostics` that compute the daily average for the given variable.
+"""
+# An average is just a sum with a normalization before output
+daily_average(short_names; output_writer = HDF5Writer()) = common_diagnostics(
+    24 * 60 * 60,
+    (+),
+    output_writer,
+    short_names;
+    pre_output_hook! = average_pre_output_hook!,
+)[1]
+"""
+    hourly_maxs(short_names...; output_writer = HDF5Writer())
+
+
+Return a list of `ScheduledDiagnostics` that compute the hourly max for the given variables.
+"""
+hourly_maxs(short_names...; output_writer = HDF5Writer()) =
+    common_diagnostics(60 * 60, max, output_writer, short_names...)
+
+"""
+    hourly_max(short_names; output_writer = HDF5Writer())
+
+
+Return a `ScheduledDiagnostics` that computse the hourly max for the given variable.
+"""
+hourly_max(short_names...; output_writer = HDF5Writer()) =
+    common_diagnostics(60 * 60, max, output_writer, short_names)
+
+"""
+    hourly_mins(short_names...; output_writer = HDF5Writer())
+
+
+Return a list of `ScheduledDiagnostics` that compute the hourly min for the given variables.
+"""
+hourly_min(short_names...; output_writer = HDF5Writer()) =
+    common_diagnostics(60 * 60, min, output_writer, short_names...)
+"""
+    hourly_mins(short_names...; output_writer = HDF5Writer())
+
+
+Return a `ScheduledDiagnostics` that computes the hourly min for the given variable.
+"""
+hourly_min(short_names; output_writer = HDF5Writer()) =
+    common_diagnostics(60 * 60, min, output_writer, short_names)[1]
+
+"""
+    hourly_averages(short_names...; output_writer = HDF5Writer())
+
+
+Return a list of `ScheduledDiagnostics` that compute the hourly average for the given variables.
+"""
+
+# An average is just a sum with a normalization before output
```

Comment:
```suggestion
# An average is just a sum with a normalization before output
"""
```
Newlines between doc strings and their functions are not allowed. It looks like there may be a bunch of these, let's apply elsewhere, too.